### ISSUE TITLE
2.0.0 - Getting IllegalArgumentException when wrapping a Task<Void> into a Completable

### ISSUE THREAD

**Issue by nihk** [ashdavies/rx-tasks#19]
I was using 1.2.0 previously but after upgrading to 2.0.0 I get an IAE when wrapping a `Task<Void>` in a Completable. Not sure if other Source types are affected. Anyway, here's my code that worked fine in 1.2.0 but crashes in 2.0.0:

```
val aFirestoreInsertionTask: Task<Void> = ...

RxTasks.completable(aFirestoreInsertionTask)
            .subscribeOn(Schedulers.io())
            .observeOn(AndroidSchedulers.mainThread())
            .subscribe(...)

```
  
```
  java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter void
        at io.ashdavies.rx.rxtasks.CompletableEmitterSuccessListener.onSuccess(CompletableEmitterSuccessListener.kt)
        at io.ashdavies.rx.rxtasks.CompletableEmitterSuccessListener.onSuccess(CompletableEmitterSuccessListener.kt:6)
        at com.google.android.gms.tasks.zzj.run(Unknown Source)
        at android.os.Handler.handleCallback(Handler.java:746)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5443)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
```

**Comment by tbsandee** [comment]
This is fixed by the merged PR #17, just waiting on a release.

**Comment by ashdavies** [comment]
Should now be available on maven central
